df.units.xml

```diff
@@ -753,15 +753,16 @@
 
         <compound name='last_hit' type-name='history_hit_item'/>
 
-        <int32_t name='riding_item_id' ref-target='item' since='v0.34.08'/>
-
         <stl-vector name='inventory' pointer-type='unit_inventory_item'/>
 
         <stl-vector name='owned_items' type-name='int32_t' ref-target='item'/>
         <stl-vector name='traded_items' comment='items brought to trade depot' type-name='int32_t' ref-target='item'/>
         <stl-vector name='owned_buildings' pointer-type='building'/>
         <stl-vector name='corpse_parts' comment='entries remain even when items are destroyed' type-name='int32_t' ref-target='item'/>
 
+        <int32_t name='riding_item_id' ref-target='item' since='v0.34.08'/>
+        <int32_t/>
```

Comment:
Would be nice to name this as `unk_#` and tag with `since='v50.01'` (if that's right), but I know a lot of new v50 fields have had these things skipped